repack_st_arch.csh

```diff
@@ -124,10 +124,10 @@ endif
 set do_forcing     = 'true'
 # > > > WARNING; if restarts fails when $mm-01 is a Monday, turn off the pre_clean flag,
 #                in order to preserve what's in rest/YYYY-MM.
-set do_restarts    = 'true'
+set do_restarts    = 'false'
 set do_obs_space   = 'false'
 set do_history     = 'true'
-set do_state_space = 'true'
+set do_state_space = 'false'
```

Comment:
I need help understanding when these should be true or false.

Reply:
long term (e.g. not before the runs are restarted):  if more than a single variable has to change to handle some situation, then it would be good to try to collect them together in a block at the top, to try to prevent errors when scattered variables are changed in multiple places in a script and possibly not all changed back.  (again, trying to minimize chances for errors.  forgetting to reset all required variables is an easy one to make if you're interrupted mid-change for any reason.)

Reply:
These displayed variables only change (as they are here) in unusual circumstances,
and need to be independent variables for more precise control.
Are there other variables that concern you?

Reply:
i'm remembering that often a error turns out to be that variables were changed to handle some situation and then forgotten and not changed back.  i'm trying to brainstorm on how to minimize the chances of this going forward.  i guess having multiple eyes review all changes is trying to address this.  maybe the process can be that we change any needed variables for unusual situations, and then to continue we revert (checkout?) the script(s) from the git version to sure no change is forgotten to be undone?  if any changes are made that should stay, those should be committed.  then a 'git diff' can show us anything that changed for an unusual circumstance?  something like that.